test/suites/smoke-test-common-solo/test-ethereum-token-transfers.ts

```diff
@@ -0,0 +1,214 @@
+import "@tanssi/api-augment/dancelight";
+
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
+import type { ApiPromise } from "@polkadot/api";
+import type { EthereumTokenTransfersNativeTokenTransferred } from "@polkadot/types/lookup";
+import { hexToU8a } from "@polkadot/util";
+import { encodeAddress } from "@polkadot/util-crypto";
+import { Interface } from "ethers";
+import { ETHEREUM_MAINNET_SOVEREIGN_ACCOUNT_ADDRESS, SEPOLIA_SOVEREIGN_ACCOUNT_ADDRESS } from "utils";
+
+const SS58_FORMAT = 42;
+
+let BLOCKS_AMOUNT_TO_CHECK = 1;
+// For debug purposes only, specify block here to check it
+const BLOCK_NUMBER_TO_DEBUG = 1623043;
```

Comment:
remember to set this back, also BLOCKS_AMOUNT_TO_CHECK

```suggestion
const BLOCK_NUMBER_TO_DEBUG = undefined;
```